### ISSUE TITLE
A Gentle Introduction to Early Stopping to Avoid Overtraining Deep Learning Neural Network Models

### ISSUE THREAD

**Issue by ikedaosushi** [ikedaosushi/tech-news#9220]
A Gentle Introduction to Early Stopping to Avoid Overtraining Deep Learning Neural Network Models<br>
Too little training will mean that the model will underfit the train and the test sets. Too much training will mean that the model will overfit the training dataset and have poor performance on the test set.<br>
https://ift.tt/2AZ4PSy
